src/Unity/Helper/Unity.Data.cs

```diff
@@ -10,23 +13,80 @@ public partial class Unity
     public Module UnityPlayer { get; private set; }
     public SceneManager Scenes { get; private set; }
 
+    private string _dataFolder;
+    public string DataFolder
```

Comment:
please explain why this is necessary (ideally with examples).

Reply:
For Prince of Persia: The Lost Crown's default version, the .exe is `TheLostCrown.exe`, the mono module is `GameAssembly.dll`, and the data folder is `TheLostCrown_Data`.

For the Ubisoft+ version of the game, the .exe is `TheLostCrown_plus.exe`, the mono module is `GameAssembly_plus.dll`, and the data folder is still `TheLostCrown_Data`.

The code currently infers the name of the data folder from the main module (`TheLostCrown_plus.exe`), so it looks for a folder which does not exist (`TheLostCrown_plus_Data`). This lets us override the folder it looks for in case it is different.

Reply:
![image](https://github.com/just-ero/asl-help/assets/8567231/33adc96b-2a56-4180-90f5-f83fc4a7b529)